library/std/src/os/windows/fs.rs

```diff
@@ -472,10 +472,14 @@ pub trait MetadataExt {
     #[unstable(feature = "windows_by_handle", issue = "63010")]
     fn file_index(&self) -> Option<u64>;
 
-    /// Returns the change time, which is the last time file metadata was changed, such as
-    /// renames, attributes, etc
+    /// Returns the value of the `ChangeTime{Low,High}` fields of this metadata.
```

Comment:
Separating it into `low` and `high` is an implementation detail that may differ in future structs. E.g. in some cases a `LARGE_INTEGER` [could be used instead](https://learn.microsoft.com/en-us/windows-hardware/drivers/ddi/ntifs/ns-ntifs-_file_stat_information#syntax).

I'd rather we just say `ChangeTime` and assume that anyone looking at the underlying structs can easily figure out what that means.

Reply:
@ChrisDenton Would you like me to fix some of the other docs in this file as well? For example, `nFileSize{High,Low}`, `nFileIndex{Low,High}`

Reply:
I mean, if you don't mind. But please don't feel like you have to.

Reply:
Sure I don't mind

Reply:
Thanks!